airdrops/index_v2.json

```diff
@@ -0,0 +1,326 @@
+            "new_asset_data": {
+                "asset_type": "EVM_TOKEN",
+                "address": "0xCa14007Eff0dB1f8135f4C25B34De49AB0d42766",
+                "name": "Starknet",
+                "symbol": "STRK",
+                "chain_id": 1,
+                "decimals": 18,
+                "coingecko": "starknet"
+            }
+        },
+        "optimism_4": {
+            "csv_path": "airdrops/optimism_4.csv",
+            "csv_hash": "01ca439a24973b1cf1c3335b2c4b4db3e9c8a68ad105ee11b9eec2450fdf7499",
+            "asset_identifier": "eip155:10/erc20:0x4200000000000000000000000000000000000042",
+            "url": "https://app.optimism.io/airdrops/4",
+            "name": "OP",
+            "icon": "optimism.svg",
+            "cutoff_time": 1739491200
+        },
+        "degen2_season1": {
+            "csv_path": "airdrops/degen2_season1.csv",
+            "csv_hash": "708ae1fcbe33a11a91fd05e1e7c4fa2d514b65cb1f8d3e4ce3556e7bdd8af2f5",
+            "asset_identifier": "eip155:8453/erc20:0x4ed4E862860beD51a9570b96d89aF5E1B0Efefed",
+            "url": "https://www.degen.tips/airdrop2/season1",
+            "name": "DEGEN",
+            "icon": "degen.svg",
+            "icon_path": "airdrops/icons/degen.svg",
+            "cutoff_time": 1711843200,
+            "new_asset_data": {
+                "asset_type": "EVM_TOKEN",
+                "address": "0x4ed4E862860beD51a9570b96d89aF5E1B0Efefed",
+                "name": "Degen",
+                "symbol": "DEGEN",
+                "chain_id": 8453,
+                "decimals": 18
+            }
+        },
+        "omni": {
+            "csv_path": "airdrops/omni.csv",
+            "csv_hash": "0e0d5520e9c18bdc4d688680deb0389e48e9a1cf7af0874696d078f6e608252d",
+            "asset_identifier": "eip155:1/erc20:0x36E66fbBce51e4cD5bd3C62B637Eb411b18949D4",
+            "url": "https://claims.omni.network/",
+            "name": "OMNI",
+            "icon": "omni.svg",
+            "icon_path": "airdrops/icons/omni.svg",
+            "new_asset_data": {
+                "asset_type": "EVM_TOKEN",
+                "address": "0x36E66fbBce51e4cD5bd3C62B637Eb411b18949D4",
+                "name": "Omni Network",
+                "symbol": "OMNI",
+                "chain_id": 1,
+                "decimals": 18
+            }
+        },
+        "eigen":{
+            "asset_identifier":"EIGEN_TOKEN_PRE_RELEASE",
+            "url":"https://eigenfoundation.org",
+            "api_url":"https://claims.eigenfoundation.org/clique-eigenlayer-api/campaign/eigenlayer/credentials?walletAddress={address}",
+            "amount_path":"data/pipelines/tokenQualified",
+            "name":"EIGEN",
+            "icon": "eigenlayer.png",
+            "icon_path":"airdrops/icons/eigenlayer.png",
+            "new_asset_data":{
+                "asset_type":"OTHER",
+                "name":"Eigen",
+                "symbol":"EIGEN"
+            },
+            "cutoff_time": 1725667200
```

Comment:
we already have `frontend/app/public/assets/images/protocols/eigenlayer.png` so new icon_path can be avoided. Also in the rotki repo PR there is also `asset_identifier` and `decimals` in `new_asset_data`, if those are not needed then can we remove them from the test? It confused me